### ISSUE TITLE
Consolidate utility functions for MediaInfo in a single location

### ISSUE THREAD

**Issue by kqyang** [google/shaka-packager#373]
Right now MediaInfo related utility functions are duplicated in [MPD code](https://github.com/google/shaka-packager/blob/634ee65663bd2eae1745db31f111e732529b97b3/packager/mpd/base/mpd_utils.cc) and in [HLS code](https://github.com/google/shaka-packager/blob/30b5fdf19fcab52e1089989677ea7a67388cafdf/packager/hls/base/media_playlist.cc).

We should move the definitions to a common location usable by both MPD and HLS code.

**Comment by rkuroiwa** [comment]
Is this a low hanging fruit? Is it a better use of resource to come up with a unified IR for generating both manifests?

**Comment by kqyang** [comment]
Yes, we should prioritize the work on unified IR (https://github.com/google/shaka-packager/issues/375) if there is resource.

On the other hand, it may give us a better idea of how the unified IR should look like by solving this issue.

In any case, this is a code health problem in the current code base.
